frontend/helpers/revisions.ts

```diff
@@ -0,0 +1,11 @@
+import { get } from 'lodash';
+import memoizeOne from 'memoize-one';
+
+export const getActiveRevisions = memoizeOne((revisions) => {
+	return revisions.filter((revision) => !revision.void);
```

Comment:
Instead of doing this, shouldn't we just make sure it utilises the 2 different APIs provided (You don't get the void revisions if you don't add the query string) for when we need the void revisions or not?

In reality no one should need to know about void revisions other than the revisions dialog in the teamspace view. This feels like we're doing more processing on the front end and it's pretty prone to error in the future...

Reply:
I will use this function only for the case that you are noticed:

>  Active revision no longer bold if it's not the latest (we need to filter to ignore void)